src/sql/workbench/services/dialog/browser/wizardModal.ts

```diff
@@ -191,6 +193,8 @@ export class WizardModal extends Modal {
 				dialogPane.hide();
 			}
 		});
+
+		status(localize('wizardPageChangedMessage', "Step {0}: {1}", index + 1, pageToShow.title));
```

Comment:
So as I was mentioning - I really don't like doing this. If we ever update the text on the wizard page we're going to have to remember to update this as well. Instead I would suggest adding another property somewhere that the step text is retrieved from and then use that for both the DialogComponent and here. Where exactly that text lives I'll leave up to you - but having it on the DialogPane and then having the DialogComponent consume that seems reasonable.